### DISCUSSION TITLE
trying to transfer files from my old Retronas HD to a newly configured Retronas

### DISCUSSION THREAD

**Discussion by rester159** [retronas/discussions#38] [Q&A]
long story short...

I had set up my retronas with a raspberry pi and a 18tb HD - everything was running great.

Since I was running out of space, I took out the old HD, created a RAID with 4 new drives, and configured retronas from scratch. Now the issue is that I have all my data in the old HD and would like to transfer everything into the RAID.

The RAID is mounted on /storage per the instructions. When I connect my old HD to the raspberry pi, it is trying to also mount it on /storage, which I guess it doesn't like. I tried to mount it to a different directory which I created ("__old HD") but I get this

mount: /old HD: mount(2) system call failed: File too large.

Maybe I am doing this totally wrong and there is a simpler way to transfer my old files to the new HD. I need to somehow find a way to 'drag and drolp' the old files into the new HD but I'm running out of ideas.

**Comment by sairuk** [comment]
migrated to a discussion

Is your RetroNAS host debian install 32bit or 64bit? you should be using 64bit for such a large drive

What is the full output of mounting verbosely e.g. `mount -vv`

What is the filesystem you are getting that error with and is it a supported fs on the new install? `cat /proc/filesystems`

**Reply by rester159** [reply]
32 bit
filesystem is EXT4 (same as the RAID)
FWIW, I have mounted a RAID 5 that is composed of 4 disks (18, 18, 5, 5) and it works...

mount -vv

/dev/mmcblk0p2 on / type ext4 (rw,noatime,nodiratime)
devtmpfs on /dev type devtmpfs (rw,relatime,size=3879380k,nr_inodes=74987,mode=755)
proc on /proc type proc (rw,relatime)
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000)
tmpfs on /run type tmpfs (rw,nosuid,nodev,size=1617700k,nr_inodes=819200,mode=755)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k)
cgroup2 on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime,nsdelegate,memory_recursiveprot)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
bpf on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=31,pgrp=1,timeout=0,minproto=5,maxproto=5,direct)
mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime)
sunrpc on /run/rpc_pipefs type rpc_pipefs (rw,relatime)
debugfs on /sys/kernel/debug type debugfs (rw,nosuid,nodev,noexec,relatime)
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime)
fusectl on /sys/fs/fuse/connections type fusectl (rw,nosuid,nodev,noexec,relatime)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /tmp type tmpfs (rw,relatime)
/dev/mmcblk0p1 on /boot type vfat (rw,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=ascii,shortname=mixed,errors=remount-ro)
/dev/mmcblk0p2 on /var/folder2ram/var/log type ext4 (rw,noatime,nodiratime)
folder2ram on /var/log type tmpfs (rw,relatime)
/dev/mmcblk0p2 on /var/folder2ram/var/tmp type ext4 (rw,noatime,nodiratime)
folder2ram on /var/tmp type tmpfs (rw,relatime)

cat /proc/filesystems

nodev   sysfs
nodev   tmpfs
nodev   bdev
nodev   proc
nodev   cgroup
nodev   cgroup2
nodev   cpuset
nodev   devtmpfs
nodev   configfs
nodev   debugfs
nodev   tracefs
nodev   securityfs
nodev   sockfs
nodev   bpf
nodev   pipefs
nodev   ramfs
nodev   rpc_pipefs
nodev   devpts
        ext3
        ext2
        ext4
        vfat
        msdos
nodev   nfs
nodev   nfs4
nodev   autofs
        f2fs
nodev   mqueue
nodev   pstore
        fuseblk
nodev   fuse
nodev   fusectl
nodev   overlay

/dev/mmcblk0p2 on /var/folder2ram/var/lib/openmediavault/rrd type ext4 (rw,noatime,nodiratime)
folder2ram on /var/lib/openmediavault/rrd type tmpfs (rw,relatime)
/dev/mmcblk0p2 on /var/folder2ram/var/spool type ext4 (rw,noatime,nodiratime)
folder2ram on /var/spool type tmpfs (rw,relatime)
/dev/mmcblk0p2 on /var/folder2ram/var/lib/rrdcached type ext4 (rw,noatime,nodiratime)
folder2ram on /var/lib/rrdcached type tmpfs (rw,relatime)
/dev/mmcblk0p2 on /var/folder2ram/var/lib/monit type ext4 (rw,noatime,nodiratime)
folder2ram on /var/lib/monit type tmpfs (rw,relatime)
/dev/mmcblk0p2 on /var/folder2ram/var/cache/samba type ext4 (rw,noatime,nodiratime)
folder2ram on /var/cache/samba type tmpfs (rw,relatime)
tmpfs on /run/user/0 type tmpfs (rw,nosuid,nodev,relatime,size=808848k,nr_inodes=202212,mode=700)
/dev/md127 on /storage type ext4 (rw,relatime,stripe=384,x-parent=1b060b3f:c5acf3f2:9c156df2:342ea8ab)

**Reply by sairuk** [reply]
you'll need a 64bit os to handle that drive size over (i presume) usb we've been over these problems a few times now which is why we recommend 64bit

* https://github.com/danmons/retronas/wiki/Supported-hardware
* https://github.com/danmons/retronas/issues/59
* https://forums.raspberrypi.com/viewtopic.php?t=330024

**Comment by rester159** [comment]
Ok cool I will reformat. As related (?) question, I created a RAID 5 in
cockpit with 4 drives (18, 18, 5 , 5) for a theoretical total of 46 - but
cockpit shows the size of the RAID as 15tb. Is that Related to the 32/64
bit issue?

**Reply by sairuk** [reply]
yep i would say that is related, once you upgrade you should be able to see the full spec of your raid. Cockpit is probably doing some rounding for output and showing 15TB as the theoretical max on 32bit is 16TB

**Comment by rester159** [comment]
Nice thanks. Will give it a shot.
